cartridge-cli.lua

```diff
@@ -1985,7 +2002,9 @@ local function pack_rpm(source_dir, dest_dir, name, release, version, opts)
         {'FILEDIGESTS', 'STRING_ARRAY', fileinfo.filedigests},
         {'FILELINKTOS', 'STRING_ARRAY', fileinfo.filelinktos},
         {'RPMVERSION', 'STRING', '4.11.3'},
```

Comment:
This feature was implemented in https://github.com/rpm-software-management/rpm/commit/91aa0786cf3b2e34de01c586427952de6d0d9b40 "4.14.0-rc1"

I suggest you to bump this version... I'm not sure that it can't cause any conflicts

Reply:
This flag is optional, so I suggest removing it from header. It just confuses.